Missing null check in DecimalPipe

**Description**

1. It's possible to set properties `FhiDiagramOptions.decimals` or `FhiDiagramOptions.unit.decimals` to `null`, but the decimal implementation doesn't support it!
2. There is also a problem concerning how decimals are set: they should only be truncated, not padded with zeros! 


**Repro steps**

1. Create test data containing `null`
2. Test in Demo-app

**Stacktrace**

```
global-error-handler.ts:15 Error: NG02100: InvalidPipeArgument: '1.null-null is not a valid digit info' for pipe 'DecimalPipe'
    at invalidPipeArgumentError (common.mjs:4053:12)
    at DecimalPipe.transform (common.mjs:4856:19)
    at pureFunction2Internal (core.mjs:28135:65)
    at Module.ɵɵpipeBind2 (core.mjs:28356:9)
    at FhiAngularHighchartsComponent_ng_template_13_tr_5_ng_container_1_td_2_ng_container_1_Template (folkehelseinstituttet-angular-highcharts.mjs:429:103)
```

--- a/projects/fhi-angular-highcharts/CHANGELOG.md
+++ b/projects/fhi-angular-highcharts/CHANGELOG.md
@@ -2,6 +2,7 @@
 
 > Apr 29, 2024
 
+* :bug: **Bugfix** Add null check for `FhiDiagramOptions.decimals` and `FhiDiagramOptions.unit.decimals`
 * :tada: **Enhancement** Set `minFractionDigits` in `digitsInfo` to `0` (DecimalPipe), since padding the number with extra zeros actually changes the number.
 
 ## 3.1.0

--- a/projects/fhi-angular-highcharts/CHANGELOG.md
+++ b/projects/fhi-angular-highcharts/CHANGELOG.md
@@ -1,4 +1,10 @@
-# 3.1.0
+# Unreleased
+
+> Apr 29, 2024
+
+* :tada: **Enhancement** Set `minFractionDigits` in `digitsInfo` to `0` (DecimalPipe), since padding the number with extra zeros actually changes the number.
+
+## 3.1.0
 
 > Apr 22, 2024
 

--- a/projects/fhi-angular-highcharts/src/lib/fhi-angular-highcharts.component.ts
+++ b/projects/fhi-angular-highcharts/src/lib/fhi-angular-highcharts.component.ts
@@ -192,6 +192,9 @@ export class FhiAngularHighchartsComponent implements OnChanges {
     // Currently only support for one unit
     const decimals = unit?.length >= 0 ? unit[0].decimals : this.allDiagramOptions.decimals; // Temporary fallback before this.allDiagramOptions.decimals deprecates in v5
 
+    if (decimals === null) {
+      return;
+    }
     if (decimals > 14) {
       console.warn('Max decimal places is 14 due to loss of precision at runtime!');
     } else if (decimals >= 0) {

--- a/projects/fhi-angular-highcharts/src/lib/fhi-angular-highcharts.component.ts
+++ b/projects/fhi-angular-highcharts/src/lib/fhi-angular-highcharts.component.ts
@@ -195,10 +195,10 @@ export class FhiAngularHighchartsComponent implements OnChanges {
     if (decimals === null) {
       return;
     }
-    if (decimals > 14) {
-      console.warn('Max decimal places is 14 due to loss of precision at runtime!');
-    } else if (decimals >= 0) {
+    if (decimals >= 0) {
       this.digitsInfo = `1.0-${decimals}`;
+    } else if (decimals > 14) {
+      console.warn('Max decimal places is 14 due to loss of precision at runtime!');
     }
   }
 

--- a/projects/fhi-angular-highcharts/src/lib/fhi-angular-highcharts.component.ts
+++ b/projects/fhi-angular-highcharts/src/lib/fhi-angular-highcharts.component.ts
@@ -193,10 +193,9 @@ export class FhiAngularHighchartsComponent implements OnChanges {
     const decimals = unit?.length >= 0 ? unit[0].decimals : this.allDiagramOptions.decimals; // Temporary fallback before this.allDiagramOptions.decimals deprecates in v5
 
     if (decimals > 14) {
-      this.digitsInfo = '1.14-14';
       console.warn('Max decimal places is 14 due to loss of precision at runtime!');
     } else if (decimals >= 0) {
-      this.digitsInfo = `1.${decimals}-${decimals}`;
+      this.digitsInfo = `1.0-${decimals}`;
     }
   }
 

